server/server.go

```diff
@@ -106,7 +108,7 @@ type Server struct {
 	StatsCloser                   io.Closer
 	Locker                        locking.Locker
 	ApplyLocker                   locking.ApplyLocker
-	VCSEventsController           *events_controllers.VCSEventsController
+	VCSPostHandler                sqs.VCSPostHandler
```

Comment:
I think it'd be better if we defined this interface in the `events` package so that we don't have to modify the reference when we sync with upstream.

Reply:
Originally thought the same, but I think it's standard to define interface with the new reader:
https://github.com/lyft/atlantis/pull/187#discussion_r808321569

Reply:
Gotcha! Thanks for the reference 👍